packages/react/src/combobox/ComboboxBase/ComboboxButtons.tsx

```diff
@@ -7,7 +7,7 @@ import { focusStyles } from '../../box';
 export const ComboboxClearButton = forwardRef<
 	HTMLButtonElement,
 	ButtonHTMLAttributes<HTMLButtonElement>
->(function ComboboxClearButton({ disabled, ...props }, ref) {
+>(function ComboboxClearButton(props, ref) {
```

Comment:
Fixed bug where the dropdown trigger button and clear button were not being marked or shown as disabled